server/dao/order.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const orderCollectionName = "orders";
+
+// --------------
+// Create Order
+// --------------
+
+exports.createOrder = async (db, clientId, products, status, totalPrice) => {
+  const newOrder = {
+    clientId: clientId.toString(),
+    products: products.toString(),
+    status: status.toString(),
+    totalPrice: totalPrice.toString(),
+  };
+  const res = await db.collection(orderCollectionName).insertOne(newOrder);
```

Comment:
Don't do async await here, let this method only return the promise that will be handled in the handler